src/hardware/vga_s3.cpp

```diff
@@ -700,9 +700,14 @@ void filter_s3_modes_to_oem_only()
 	case 8192 * 1024: dram_size = mb_8; break;
 	}
 	auto mode_not_allowed = [&](const VideoModeBlock &m) -> bool {
-		// Allows all the standard VESA modes, which start prior to 0x120
-		if (m.mode < 0x120)
+		// Allow all the standard VESA modes, which start prior to 0x120,
+		// except for 320x200x15/16/24 which were rarely supported until late 90s.
```

Comment:
Suggest moving the `// except for ..` comment down above the `if` line where the exception is made.